rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,56 @@
+# Upgrade rook-ceph to v1.12.x
```

Comment:
Should this still exist? Same from 13 and 14.

Reply:
Yeah, what do people think? @Xartos @robinAwallace @aarnq @viktor-f

Reply:
Its now done, does it look correct?